### ISSUE TITLE
Add support for .npmrc configured mirror of binaries

### ISSUE THREAD

**Issue by joachimprinzbach** [aerokube/cypress-moon#11]
It would be great to be able to mirror the binary through a .npmrc setting. 

`
CYPRESS_MOON_BINARY=mymirror.com/cypress-moon/download
`

Adding sth. like this to the lib/index.js could be a starting point
```js
const binaryUrlOverride = process.env.npm_config_CYPRESS_MOON_BINARY;

const url = binaryUrlOverride ?
    `${binaryUrlOverride}/${moonVersion}/${binName}` :
    `https://github.com/aerokube/moon/releases/download/${moonVersion}/${binName}`;
```

Highlighted changes: 
- read from `npm_config_CYPRESS_MOON_BINARY`
- add `moonVersion` and `binName` to the binaryUrlOverride as well

What do you guys think?

**Comment by vania-pooh** [comment]
@joachimprinzbach this is in fact already available in `@aerokube/cypress-moon@next` version of this package. Not yet released. You can completely override download URL or use local path. See: https://github.com/aerokube/cypress-moon/pull/8 Will release this soon.

**Comment by joachimprinzbach** [comment]
Yeah I know, I already used this feature as I have the beta version in my package.json. 
However, it is not working with .npmrc and I would have to add the version to binaryUrl instead of using the version from the cypress-moon package. In my opinion the change I suggested above solves those 2 problems.

**Comment by vania-pooh** [comment]
@joachimprinzbach seems to be a reasonable modification. If you could suggest possible modification in current source code, this could dramatically accelerate things.
